tests/models/test_llava.py

```diff
@@ -0,0 +1,139 @@
+import pytest
+from typing import Tuple, List, Dict
+
+from enum import Enum
+from dataclasses import fields
+import torch
+from transformers import AutoTokenizer
+
+from vllm.config import VisionLanguageConfig
+
+model_and_vl_config = [
+    ("llava-hf/llava-1.5-7b-hf",
+     VisionLanguageConfig(
+         image_input_type=VisionLanguageConfig.ImageInputType.PIXEL_VALUES,
+         image_feature_size=576,
+         image_token_id=32000,
+         image_input_shape=(1, 3, 336, 336))),
+    # ("llava-hf/llava-1.5-7b-hf",
+    #  VisionLanguageConfig(
+    #      image_input_type=VisionLanguageConfig.ImageInputType.IMAGE_FEATURES,
+    #      image_feature_size=576,
+    #      image_token_id=32000,
+    #      image_input_shape=(1, 576, 1024)))
+]
+
+
+def as_dict(vision_language_config: VisionLanguageConfig) -> Dict:
```

Comment:
Doesn't `dataclasses.asdict(config)` already do this? If no, what's the difference?

Reply:
if the value is an enum or a tuple, it still remains so in asdict, while I need it to be pure string to be compatible with LLM entrypoint requirement.